doc/modules/ROOT/pages/tutorials/centrality-algorithms.adoc

```diff
@@ -0,0 +1,317 @@
+// DO NOT EDIT - AsciiDoc file generated automatically
+
+= Centrality Algorithms
```

Comment:
Missing colab link? Other page too

Reply:
true, they exist in the notebooks but got lost during conversion :thinking:

Reply:
@nvitucci do you have an idea why the cell is ignored? I cant spot the difference to other notebooks

Reply:
The way the converter works at the moment requires the `colab_type` key to be present in the Colab cell metadata (which is empty in this notebook), like this:

```
"metadata": {
    "colab_type": "text",
    "id": "view-in-github"
   }
```

Reply:
thanks that solved it. Should have looked at the raw files and not the VS Code view